k2/python/csrc/torch/v2/any.cu

```diff
@@ -70,15 +70,47 @@ void PybindRaggedAny(py::module &m) {
 
   any.def(
       "__getitem__",
-      [](RaggedAny &self, int32_t i) -> RaggedAny {
-        return self.Index(/*axis*/ 0, i);
+      [](RaggedAny &self, int32_t i) -> py::object {
+        if (self.any.NumAxes() > 2) {
+          RaggedAny ragged = self.Index(/*axis*/ 0, i);
+          return py::cast(ragged);
+        } else {
+          K2_CHECK_EQ(self.any.NumAxes(), 2);
+          const int32_t *row_split1_data = self.any.RowSplits(1).Data();
+          int32_t begin = row_split1_data[i],
```

Comment:
Shall we move row splits to CPU and then access its data. It can reduce one memory copy kernel call.